src/Tribe/Views/V2/Template_Bootstrap.php

```diff
@@ -354,21 +354,28 @@ public function filter_add_body_classes( $classes ) {
 	 * @param string $file      Complete path to include the PHP File
 	 * @param array  $name      Template name
 	 * @param object $template  Instance of the Tribe__Template
-	 * 
+	 *
 	 * @return string
 	 */
 	public function filter_template_file( $file, $name, $template ) {
-		
-		if ( 
+		if (
```

Comment:
:no_entry_sign: Spaces must be used to indent lines; tabs are not allowed

<sub>hash: b56807c5a2f82b5e78a5bdec5cd37a2a</sub>